cmd/crc/cmd/daemon_windows.go

```diff
@@ -1,14 +1,95 @@
 package cmd
 
 import (
+	"fmt"
 	"os"
 
 	"github.com/code-ready/crc/pkg/crc/api"
 	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/pkg/errors"
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/debug"
+	"golang.org/x/sys/windows/svc/eventlog"
 )
 
-func runDaemon() {
+func runDaemon() error {
 	// Remove if an old socket is present
-	os.Remove(constants.DaemonSocketPath)
-	api.RunCrcDaemonService("CodeReady Containers", false, newConfig, newMachineWithConfig)
+	_ = os.Remove(constants.DaemonSocketPath)
+	return RunDaemonService("CodeReady Containers", false)
```

Comment:
Extract as this is a name defined outside of the code.
The variable name in that case can be more descriptive about this.

Reply:
done!